src/main/python/main/ayab/engine/communication_ip.py

```
@@ -0,0 +1,215 @@
+            return True
+        except:
+            self.logger.info("Open TCP Socket faild")
+            return False
+
+    def close_socket(self):
+        if self.__sockTCP is not None:
+            try:
+                self.__sockTCP.close()
+                del self.__sockTCP
+                self.logger.info("Closing TCP Socket successful.")
+            except:
```

Ruff

Do not use bare `except`

(E722)